Error: Navigation lists must be nested in a `nav` element
Error: Current page must be indicated using `aria-current="page"`
Error: Requested subnav items were missing
Error: Styles were not requested
Error: Style changes (such as bold) must not be the only means of indicating a link is the current page
diff --git a/output/Basic Navigation with Current Page/3/f8457372-2caa-489b-9d29-16a703733b51.html b/output/Basic Navigation with Current Page/3/f8457372-2caa-489b-9d29-16a703733b51.html
index 679e096..ab592f0 100644
--- a/output/Basic Navigation with Current Page/3/f8457372-2caa-489b-9d29-16a703733b51.html	
+++ b/output/Basic Navigation with Current Page/3/f8457372-2caa-489b-9d29-16a703733b51.html	
@@ -1,10 +1,13 @@
-<ul>
-  <li><a href="/">Home</a></li>
-  <li><a href="/products" class="active">Products</a></li>
-  <li><a href="/services">Services</a></li>
-</ul>
-<style>
-  .active {
-    font-weight: bold;
-  }
-</style>
\ No newline at end of file
+<nav>
+  <ul>
+    <li><a href="/">Home</a></li>
+    <li>
+      <a href="/products" aria-current="page">Products</a>
+      <ul>
+        <li><a href="/products/frobs">Frobs</a></li>
+        <li><a href="/products/thrips">Thrips</a></li>
+      </ul>
+    </li>
+    <li><a href="/services">Services</a></li>
+  </ul>
+</nav>
\ No newline at end of file
